### DISCUSSION TITLE
Bugs/Issues

### DISCUSSION THREAD

**Discussion by mjxbenet** [edemaine/cocreate#147] [General]
1.  Sometimes a user cannot see any changes on a cocreate board until the refresh the page, and can't see changes in realtime.

**Comment by edemaine** [comment]
This sounds like #107, which seems to be an issue with redis-oplog but extremely difficult to track down, and so far I have failed. Restarting the server is the best fix, but it's difficult to know even when I should restart the server. If anyone has ideas, I'm all ears.
